src/views/clusteroverview/TopConsumersTab/TopConsumersTab.tsx

```
@@ -33,7 +33,7 @@ const TopConsumersTab: FC = () => {
           actions={{
             actions: (
               <>
-                <Link to="/monitoring/dashboards/grafana-dashboard-kubevirt-top-consumers?period=4h">
+                <Link id="view-virtualization-dashboard" to="/monitoring/dashboards/grafana-dashboard-kubevirt-top-consumers?period=4h">
```

ESLint

[error] Replace `·id="view-virtualization-dashboard"·to="/monitoring/dashboards/grafana-dashboard-kubevirt-top-consumers?period=4h"` with `⏎··················id="view-virtualization-dashboard"⏎··················to="/monitoring/dashboards/grafana-dashboard-kubevirt-top-consumers?period=4h"⏎················`

(prettier/prettier)